src/battle_script_commands.c

```diff
@@ -58,10 +58,226 @@ extern struct MusicPlayerInfo gMPlayInfo_BGM;
 
 extern const u8* const gBattleScriptsForMoveEffects[];
 
-// this file's functions
+// courtesy of doesnt
+static const s32 sExperienceScalingFactors[] = 
```

Comment:
I forgot to say this in my previous review, but I feel the comment should clarify why it's using a lookup instead of trying to replicate the formula (for documentation's sake).
Also, maybe it'd be better to have multiple values per line, to avoid scrolling too much.
Besides that, it looks good.

Reply:
also i think there should be a comment on how each entry is calculated (because i sure as hell can't find it)

Reply:
It's here: https://bulbapedia.bulbagarden.net/wiki/Experience#Gain_formula

I assumed they did it this way to avoid having to do ^2.5

Reply:
Added in bc5ba41